documentation/writing-to-vespa.html

```diff
@@ -248,166 +206,3 @@ <h2 id="performance">Performance</h2>
 </p>
 
 
-<h3 id="load-priority">Load priority</h3>
-<p>
-Read/write operations are assigned default <em>priority levels</em> -
-modify these in <em>services.xml</em> using <a href="reference/services.html#clients">load types</a>.
-Metrics are generated per load type - example:
-<pre>
-&lt;get&gt;
-  &lt;default&gt;
-    &lt;count count="1"/&gt;
-    &lt;latency average="0.125977" last="0.125977" min="0.125977" max="0.125977" count="1"/&gt;
-    &lt;not_found count="1"/&gt;
-  &lt;/default&gt;
-  &lt;foo&gt;
-    &lt;count count="1"/&gt;
-    &lt;latency average="0.25708" last="0.25708" min="0.25708" max="0.25708" count="1"/&gt;
-  &lt;/foo&gt;
-  &lt;sum&gt;
-    &lt;count count="2"/&gt;
-    &lt;latency average="0.191528" last="0.25708" min="0.125977" max="0.25708" count="2"/&gt;
-    &lt;not_found count="1"/&gt;
-  &lt;/sum&gt;
-&lt;/get&gt;
-</pre>
-</p>
```

Comment:
I agree the section is not good. but we lack a description of this feature in vespa, albeit in another form or elsewhere. please keep it for now and I can rewrite it

Reply:
We don't need this. This is reference material.